hazelcast/src/main/java/com/hazelcast/sql/impl/extract/QueryPath.java

```diff
@@ -24,14 +24,15 @@
 
 import javax.annotation.Nullable;
 import java.io.IOException;
+import java.io.Serializable;
 
 import static com.hazelcast.query.QueryConstants.KEY_ATTRIBUTE_NAME;
 import static com.hazelcast.query.QueryConstants.THIS_ATTRIBUTE_NAME;
 
 /**
  * Represent a path to the attribute within a key-value pair.
  */
-public final class QueryPath implements IdentifiedDataSerializable {
+public final class QueryPath implements IdentifiedDataSerializable, Serializable {
```

Comment:
Why `Serializable` is suddenly needed here?

Reply:
Jet uses common Java serialization and doesn't support IDS. 
It is required to serialize `MapScanMetadata` by SQL engine.

Reply:
But `MapScanMetadata` is already `IdentifiedDataSerializable`

Reply:
I removed it and the test seems to pass :)